utils/wasm-proc/src/main.rs

```diff
@@ -170,18 +74,22 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             res.optimized_size
         );
 
-        let mut module = parity_wasm::deserialize_file(file)?;
+        let mut optimizer = Optimizer::new(file)?;
 
         if !skip_stack_end {
-            let _ = gear_wasm_builder::insert_stack_end_export(&mut module)
-                .map_err(|s| log::debug!("{}", s));
+            optimizer.insert_stack_and_export();
         }
 
         if !skip_opt {
-            optimize(file, module.clone())?;
+            let code = optimizer.optimize()?;
+            let path = optimizer.optimized_file_name();
+            fs::write(path, code)?;
         }
+
         if !skip_meta {
-            optimize_meta(file, module.clone())?;
+            let code = optimizer.metadata()?;
+            let path = optimizer.metadata_file_name();
+            fs::write(path, code)?;
```

Comment:
seems optimiser doesn't store changes inside of it.
so lines 89..92 won't contain changes applied in line 84, isn't it?